### PR TITLE
fix(bas): clarify instructions for inserting voter cards

### PR DESCRIPTION
**Pull request by eventualbuddha**
Closes https://github.com/votingworks/bas/issues/79

localhost_3000_(Surface Go)

### CODE REVIEW

**Comment by beausmith** [comment]
@benadida I feel that "New Voter Card" implied that they should not insert "old voter cards". 

I think the text should be: "Insert a Voter Card"

File: apps/bas/src/screens/InsertCardScreen.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const InsertCardScreen: React.FC<Props> = ({ lockScreen }) => {
               src="/images/insert-card.svg"
               alt="Insert Card Diagram"
             />
-            <h1>Insert Voter Card</h1>
+            <h1>Insert a New Voter Card</h1>
```
Comment (beausmith) [inline apps/bas/src/screens/InsertCardScreen.tsx:29]:
```suggestion
            <h1>Insert a Voter Card</h1>
```

Comment (beausmith) [reply apps/bas/src/screens/InsertCardScreen.tsx:29]: Because I think that "New" will be confused with "old" or "used" voter cards.

**Review by beausmith — CHANGES_REQUESTED** [review]

File: apps/bas/src/screens/InsertCardScreen.tsx (hunk 1 above)
Comment (eventualbuddha) [reply apps/bas/src/screens/InsertCardScreen.tsx]: Updated copy, but didn't update the screenshot.

**Review by beausmith — APPROVED** [review]

**Comment by benadida** [comment]
Yeah, good call @beausmith. Maybe even better as `Insert a Voter Card to Encode`.

**Pull request merged**